--- extension.ts
+++ extension.ts
@@ -10,6 +10,7 @@
 	const provider = new ChatGPTViewProvider(context.extensionUri);
 
 	provider.setAuthenticationInfo({
+		xyz: config.get<string>('xyz'),
 		apiKey: config.get<string>('apiKey')
 	});
 	provider.setSettings({
@@ -59,6 +60,8 @@
 		} else if (event.affectsConfiguration('chatgpt.timeoutLength')) {
 			provider.setSettings({ timeoutLength: config.get<number>('timeoutLength') || 60 });
 		}
+		else if (event.affectsConfiguration('chatgpt.xyz')) {
+			provider.setSettings({ xyz: config.get<string>('xyz') });
 		}
 	});
 }
